index.html

```diff
@@ -178,10 +178,10 @@ <h2 id="hd"></h2>
           </div>
           </div>
         <div class="modalImage">
-          <img  alt="First Card" class="mImage" src="#" />
+          <img  alt="First Card" class="mImage" src="mobile-images/works/Portfolio2.png"/>
         </div>
         <div class="modalImageDesktop">
-          <img  alt="card-desktop" class="mImageDsk" src="#"/>
+          <img  alt="card-desktop" class="mImageDsk" src="desktop-images/desktoppopupimg.svg"/>
         </div>
         <div class="modalTextWrapper">
           <p class="modalText"></p>
```

Comment:
[optional] Kindly make a container and set it to 100% so that it encloses all your project, your project isn't flexible in mobile view.
make sure you used percentages of your widths for large features such as containers and major parents elements. this will make your project easily adjust to different screen sizes without overflowing
 this is currently how your project shows on mobile view
![image](https://user-images.githubusercontent.com/43564324/162590806-f7010820-3435-4db5-8341-7afb7b22014d.png)